### PR TITLE
Create an OperationTelemetry base abstract class for RequestTelemetry…

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
… and DependencyTelemetry

### CODE REVIEW

File: src/Core/Managed/Shared/DataContracts/RequestTelemetry.cs [hunk 1]
```diff
@@ -35,7 +36,7 @@ public RequestTelemetry()
             // Initialize required fields
             this.Id = WeakConcurrentRandom.Instance.Next().ToString(CultureInfo.InvariantCulture);
             this.ResponseCode = "200";
-            this.Success = true;
+            this.Success = null;
```
Comment (abaranch) [inline src/Core/Managed/Shared/DataContracts/RequestTelemetry.cs]: We either need to remove ResponseCode initialization or return true back

File: src/Core/Managed/Shared/Extensibility/Implementation/OperationTelemetry.cs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+        /// <summary>
+        /// Gets the object that contains contextual information about the application at the time when it handled the request.
+        /// </summary>
+        public abstract TelemetryContext Context { get; }
+
+        /// <summary>
+        /// Gets or sets the value that defines absolute order of the telemetry item.
+        /// </summary>
+        public abstract string Sequence { get; set; }
```
Comment (abaranch) [inline src/Core/Managed/Shared/Extensibility/Implementation/OperationTelemetry.cs]: Isn't that deprecated property?..

**Comment by abaranch** [comment]
Approved with suggestions

**Comment by JakubOleksy** [comment]
:shipit:

**Pull request merged**